coveralls.json

```diff
@@ -0,0 +1,11 @@
+{
+  "skip_files": [
+    "lib/alice.ex",
+    "lib/alice/state_backends/redix_pool.ex"
```

Comment:
These files are both Supervisors. There is actually a test file for `lib/alice.ex` that tests the one function that isn't part of the Supervisor spec.